libcontainer/cgroups/fs2/fs2.go

```diff
@@ -175,57 +175,41 @@ func (m *manager) Set(container *configs.Config) error {
 	if err := m.getControllers(); err != nil {
 		return err
 	}
-	var errs []error
 	// pids (since kernel 4.5)
-	if _, ok := m.controllers["pids"]; ok {
```

Comment:
I believe this function was the only user of `controllers` map. If there aren't any left, remove it altogether.

Maybe as a separate patch, describing why there's no need for this map and check.

Reply:
agree with you

Reply:
Oh, there is another user: `GetStats`!